src/components/data/DataTable.tsx

```diff
@@ -247,7 +247,8 @@ export default function DataTable<Row>({
 
   const handleKeyDown = useCallback(
     (event: KeyboardEvent, row: Row) => {
-      if (event.key === 'Enter') {
+      // Ignore events triggered in children elements
```

Comment:
This comment says what the code does, but not why. The goal here is to avoid confirming the selection if the user presses "Enter" to activate an interactive child control.